packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -12,25 +12,36 @@
 # License for the specific language governing permissions and
 # limitations under the License.
 
+%if 0%{?amzn} >= 2
+%bcond_without systemd # with
+%else
+%bcond_with systemd # without
+%global running_semaphore %{_localstatedir}/run/ecs-init.was-running
+%endif
+%global _cachedir %{_localstatedir}/cache
+%global bundled_agent_version %{version}
+
 Name:           ecs-init
 Version:        1.18.0
-Release:        1%{?dist}
-Group:          System Environment/Base
-Vendor:         Amazon.com
```

Comment:
Why do we no longer set Group or Vendor?

Reply:
https://fedoraproject.org/wiki/Packaging:Guidelines#Tags_and_Sections

Reply:
Why is that? Does that apply to all packages? It's not clear from that doc if there are fedora-specific reasons to avoid these tags or if they must not be present even in third-party packages.

Reply:
AL2 inherits the packaging guidelines from Fedora / EPEL, except for the ones we are explicitly incompatible with (such as ones requiring changes in RPM 4.14) or otherwise simply don't like (none of these are documented yet).

The Group field, I believe, is not listed in yum repodata (deprecated in favor of yum's groups in comps.xml). The Vendor field gets overwritten by our build system anyway.